packages/frontend/src/components/ResultsTable/ResultsTable.tsx

```diff
@@ -89,9 +90,13 @@ const getItemStyle = (
     ...(isDropAnimating && { transitionDuration: '0.001s' }),
 });
 
+interface ColumnProps extends HeaderGroup {
+    description?: string;
+}
```

Comment:
Have you seen our typing in `packages/frontend/src/types/react-table-config.d.ts` ? we seem to have defined the other custom properties in there.